modules/home-manager/lsd.nix

```
@@ -0,0 +1,27 @@
+{ catppuccinLib }:
+{ config, lib, ... }:
+
+let
+  inherit (config.catppuccin) sources;
+  cfg = config.catppuccin.lsd;
```

we need to include this new enable, because we don't want to install the theme to someone's system who isn't expecting it. This previously would have been gated by `config.programs.lsd.enable` anyways prior to moving to `xdg.configFile`
```suggestion

  cfg = config.catppuccin.lsd;
  enable = cfg.enable && config.programs.lsd.enable;
```